ux.symfony.com/templates/demos/live_component/infinite_scroll_2.html.twig

```diff
@@ -0,0 +1,153 @@
+                    <path class="retro"
+                          d="M1042.5 335.4 944.9 422l-51.7 47 20 18.8L995 416l68.1-55.5-20.5-25ZM24.5 332l100.4 85.2 54.1 49.3-21 20.5-73.5-64.9-79.9-67L24.5 332ZM657.9 9 650 6c-6-3-11-3-15 1a224 224 0 0 1-119 27c-24-1-52-4-65-8-7-2-26-11-33-16-5-4-13-7-16-7l-5 2c-1 2-5 3-8 3l-1.3.4-1.6.6c17.6 29 71.8 50 135.9 50 64 0 118.2-21 135.9-50Z"/>
+                    <path class="retro"
+                          d="m1042.8 331.1 24.4 29.8-70.3 57.4-83.7 73.5-24.4-22.7 54-49.3 100-88.7ZM995 416l68.1-55.5-20.5-25L945 422l-51.7 47 20 18.8L995 416ZM24.2 327.8l102.7 87.1 56.5 51.5-25.3 24.7-75.6-66.7-82.1-69 23.8-27.6ZM84.5 422l73.5 65 21-20.5-54-49.3L24.4 332 4.6 355l79.9 67.1ZM662.3 7.5l-1.8 3c-9.4 15.4-28 28.2-52.1 37.2-24.1 9-54 14.3-86.4 14.3-32.4 0-62.3-5.3-86.4-14.4-24.1-9-42.8-21.8-52.1-37l-2-3.3 3.6-1.2 1.7-.5 1.3-.4.4-.2h.5c1.2 0 2.6-.2 3.8-.6a4.3 4.3 0 0 0 1.5-.7l.5-1 6.6-2.7h.6c2.1 0 5.4 1 8.4 2.2 3.1 1.3 6.6 3.2 9.4 5.4a128 128 0 0 0 16 8.8c6.5 3.1 12.8 5.8 16 6.7 12.7 4 40.3 6.9 64.3 7.9A221 221 0 0 0 633.3 4.6c2.4-2.3 5.2-3.6 8.4-3.7a19 19 0 0 1 9.6 2.4l7.8 2.9 3.3 1.3ZM650 6c-6-3-11-3-15 1a224 224 0 0 1-119 27c-24-1-52-4-65-8-7-2-26-11-33-16-5-4-13-7-16-7l-5 2c-1 2-5 3-8 3l-1.3.4-1.6.6 1.7 2.6C407 39.2 459.8 59 522 59c62.1 0 115-19.8 134.2-47.4L658 9a1291.2 1291.2 0 0 1-7.9-3Z"/>
+                </symbol>
+            </svg>
+
+            {# Retro Tshirt Infinite Scroll collection #}
+            <twig:ProductGrid2/>
+        </twig:Browser>
+    </div>
+
+    <section class="py-5">
+        <div class="container container-xxl">
+            <div class="code-description text-center">
+                <h2>So... how does it work?</h2>
+                <p>
+                    Let's focus on two aspects. But first, <code>did you scroll</code> all the way to the end?
+                    <br>
+                    You might have missed some important details about performance... 🍿
+                </p>
+            </div>
+        </div>
+    </section>
+
+    <section class="section--alt py-5">
+        <div class="container container-xxl d-flex flex-column">
+            <twig:Code:CodeWithExplanationRow
+                filename="templates/components/ProductGrid2.html.twig"
+                lineStart="1"
+                lineEnd="10"
+                reversed
+                codeCols="7"
+                copyButton="0"
+            >
+                ## Morphin' Trick
+
+                This trick is very similar to the one used in the
+                [`Part 1/2`]({{ url('app_demo_live_component_infinite_scroll') }})
+                of this "Infinite Scroll" demo with LiveComponent.
+
+                ### Previous Results
+
+                🐯 For the previous page results, we add one fake item
+                corresponding to the last item of the previous page,
+                with the same `id`. This allows the **addition of new elements** after the existing ones.
```

Comment:
> Maybe we could go a little deeper here and explain what morphism is and what morphism does with this same id?

I think this would require a dedicated page (and i have the perfect slide for this remember ? 😆 )

Reply:
> And I think this part should be after the observer part?

Oh i remember now... we need to define why the div 'previous' / 'current' / 'next' at first, because elsewhen this is impossible to understand ... :/ 


To be honest i'm not entirely ok with the final result, and i really want to create something more interactive / steps by steps highlither, etc.. 

I'd say "this is ok" for this one, but i'm really trying a lot of things these days to "more"